packages/file-search/src/browser-only/file-search-service-impl.ts

```diff
@@ -0,0 +1,155 @@
+// *****************************************************************************
+// Copyright (C) 2025 robertjndw, TypeFox
```

Comment:
Typefox? If it's not a 1:1 copy then it's fine to write `robertjndw and others` etc.